<launch>
  <arg name="rosbag_path" default=""/>
  <arg name="image_topic_name" default=""/>
  <node pkg="rosbag" type="play" name="rosbag" args="-d 2  $(arg rosbag_path)"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" output="screen" cwd="ROS_HOME">
    <!-- <remap from="image" to="/camera/color/image_raw"/> -->
    <remap from="image" to="$(arg image_topic_name)"/>
  </node>
</launch>